src/lib/html.ts

```diff
@@ -0,0 +1,4 @@
+import htm from 'htm';
+import { createElement } from 'preact';
+
+export const html = htm.bind(createElement);
```

Comment:
There's a highly [optimized import for htm + preact](https://github.com/developit/htm#usage):

```js
import { html } from 'htm/preact';
```

Reply:
there's actually no special code in that: https://github.com/developit/htm/blob/master/src/integrations/preact/index.mjs#L14-L19

but I agree with the question of whether it's needed

Reply:
Yes indeed, and there's no impact on bundle size from what I'm seeing. Also, the library doesn't bundle Preact so there shouldn't be conflicts. I'll remove this module and only keep the dependency.

I don't know how this version is more optimized though, because the [implementation is the same](https://github.com/developit/htm/blob/master/src/integrations/preact/index.mjs). But anyways 🤷‍♀️

Reply:
Ah ben I guess we agree 😂